Package.resolved

```diff
@@ -1,16 +0,0 @@
-{
-  "object": {
-    "pins": [
-      {
-        "package": "xctest-dynamic-overlay",
-        "repositoryURL": "https://github.com/pointfreeco/xctest-dynamic-overlay",
-        "state": {
-          "branch": null,
-          "revision": "603974e3909ad4b48ba04aad7e0ceee4f077a518",
-          "version": "0.1.0"
-        }
-      }
-    ]
-  },
-  "version": 1
-}
```

Comment:
Is there a reason you deleted this file?

Reply:
Yep. The Resolved file is not needed in a Library only target and should not be checked in. Otherwise Xcodes always thinks that this exact version should be checked out on a freshly cleaned repo.

This leads to Problems when you are working with CI Systems and Apps that consumes this Library because we have there a Package.resolved as well. You indirectly pinning the version with it to the resolved version.

Reply:
Yeah, we're thinking now we should drop all Package.resolved files from our libraries except for any that have 3rd party dependencies, such as swift-identified-collections.

Reply:
Yip, that sounds like a good decision 👍

Reply:
Just a follow-up to prevent confusion in the future, but I believe we always want to check in the `Package.resolved`, for stabler CI.

We don't believe checking a `Package.resolved` file into a library affects anyone that depends on the library, and believe it is only used when working on the package directly.